README.md

```diff
@@ -1,4 +1,58 @@
-# quizard-documentation
+# Quizard App
+
+This repository contains the Frontend source code for quizard app.
+
+## Getting Started 
+
+This guide assumed you have NodeJS 16+ installed on your pc.
+
+## Clone this repository
+
+git clone https://github.com/CeoFred/quizard-frontend.git
+
+## Set up environment variables
+
+Use the content of .env.example to create a .env file. Alternatively, you can run the command to perform that operation
+
+```bash
+cp .env.example .env
+```
+
+### Install all dependencies 
+
+```bash
+# using npm
+npm install
+
+# using yarn
+yarn install
+```
+
+Then, run the development server or app:
+
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+
+## Learn More
+
+To learn more about Next.js, take a look at the following resources:
+
+- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+
+## Project Scope
```

Comment:
I think this "Project Scope" and its content should come before "Getting Started"

Reply:
I dont agree with you, firstly this is the frontend of the app, not the whole product, in the sense that anyone that has access to the repo is a FE person that wants to quick start with the project. I dont know if you understand me shaa

Reply:
I understand. It's cool.